tests/core_test.py

```diff
@@ -536,10 +538,16 @@ def test_basic_mul_div(spectype, factor):
 
     spec = get_spectrum(spectype)
 
-    mult = spec * factor
-    assert np.allclose(mult.counts_vals, factor * spec.counts_vals)
-    assert np.allclose(mult.counts_uncs, factor * spec.counts_uncs)
-    assert mult.livetime is None
+    mult_left = spec * factor
+    assert np.allclose(mult_left.counts_vals, factor * spec.counts_vals)
+    assert np.allclose(mult_left.counts_uncs, factor * spec.counts_uncs)
+    assert mult_left.livetime is None
+
+    mult_right = factor * spec
+    assert np.allclose(mult_right.counts_vals, factor * spec.counts_vals)
+    assert np.allclose(mult_right.counts_uncs, factor * spec.counts_uncs)
+    assert mult_right.livetime is None
```

Comment:
Should this be changed to `np.isnan` like below? Otherwise I find `pd.isnull` catches more `null` cases.

```python
In [11]: pd.isnull(None)
Out[11]: True

In [12]: np.isnan(None)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-12-f53008f0bd14> in <module>()
----> 1 np.isnan(None)

TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

In [13]: pd.isnull(np.nan)
Out[13]: True
```

That said `np.isnan` is a bit faster for numeric arrays:

```python
In [20]: null_arr = np.array([np.nan, 1.] * 10000)

In [21]: %timeit np.isnan(null_arr)
4.07 µs ± 19 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [22]: %timeit pd.isnull(null_arr)
14.6 µs ± 36.2 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Reply:
I'm a bit confused about that? You want to check for `pd.isnull(mult_right.livetime)` or `pd.isnull(mult_right.counts_vals - factor * spec.counts_vals)`? I think at this point the `livetime` has to be `None` by construction.

Reply:
Oh right, okay this is good to go.